pkg/apis/clusteroperator/v1alpha1/defaults.go

```diff
@@ -20,12 +20,16 @@ import (
 	"k8s.io/apimachinery/pkg/runtime"
 )
 
+const defaultSSHUser = "root"
```

Comment:
Do we really want the default SSH user to be root? That seems unnecessarily dangerous.

Reply:
Via Kenny this was what they're using today, but we could go without a default instead and make it a required field, and you have the thus opt into using root.

Reply:
Switching it to clusteroperator, as that's what I'd like to use eventually when we can create/config the user.

Reply:
Updated.